### PR TITLE
Permutation & Subsequence of String and all paths of M x N grid

### PR DESCRIPTION
**Pull request by navneet15**

### CODE REVIEW

**Comment by ashokdey** [comment]
Hello @navneet15, it's great to see your PR for a new problem. However, I would request you to follow the folder naming convention. 

Please change the folder name from **getStringPermutaion** to **get_string_permutations**

Thanks a lot!

**Comment by rama41222** [comment]
@navneet15 Good job! It will be helpful if you can mention the time complexity as a comment. Please refer this [file](https://github.com/knaxus/problem-solving-javascript/blob/master/src/_Problems_/merge-two-sorted-arrays/index.js)

File: src/_Problems_/get-mazePath/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+//======================================Problem Statement=============================================
+// --->> Print all possible path to reach the end of the GRID/MAZE (N x N) from starting  point to ending point
+// --->> One horizontal move will be represented by H and one vertical move will be represented by V
+// --->> Complexity =  Complexity will be exponential as there are many overlapping solutions
```
Comment (TheSTL) [inline src/_Problems_/get-mazePath/index.js]: Complexity for this should be :
2^n where n = No of cells between start point to end point.
Example: 
- For (0,0) -> (2,2) , n= 9
- For (0,0) -> (0,2) , n= 3

Comment (TheSTL) [inline src/_Problems_/get-mazePath/index.js]: @navneet15 @ashokdey 
Complexity = 2^n where n = No of cells between start point to end point.

Example: 
- For (0,0) -> (2,2) , n= 9
- For (0,0) -> (0,2) , n= 3

**Pull request merged**